package.json

```diff
@@ -29,14 +29,17 @@
     "express-partials": "0.3.0",
     "express-session": "1.15.5",
     "graphql": "0.10.5",
+    "isomorphic-fetch": "2.2.1",
+    "node-fetch": "1.7.3",
     "morgan": "1.8.2",
     "nodemon": "1.11.0",
     "passport": "0.3.2",
-    "passport-google-oauth20": "1.0.0"
+    "passport-google-oauth20": "1.0.0",
+    "whatwg-fetch": "2.0.3"
   },
   "devDependencies": {
     "chai": "4.1.0",
-    "dotenv": "^4.0.0",
+    "dotenv": "4.0.0",
```

Comment:
@michaelmurray6298 I know you didn't author this line, but I believe we need this package to be in `dependencies` since we do rely on it in production.